web/src/utils/getSentryUuid.ts

```diff
@@ -0,0 +1,12 @@
+export const getSentryUuid = () => {
```

Comment:
Hmm, I think this is a borderline GDPR problem, as we now have an "identifier" for a specific user.. But as long as we're not collecting more data it's probably okay, just want to make sure we keep it in mind going forward :)

Reply:
Is it really though, it's completely anonymous and random?

We need something unique otherwise we won't be able to get a user count to correlate errors with.

Reply:
As far as I understand, it's not actually anonymous but pseudonymised data, since the identifier is for the same person and we keep adding data on this user over time (for each crash report).

> You UUID is an indirect identifier that allows you to link all the actions of a single user together. You don't know the "name" of you user anymore, but with all that info you can single out one specific user and still analyse their activity: you are in a case of identifiable user.

With that said I still don't think it's a biggie and I personally think the benefits win here, just want to make sure we're aware of the "consequences" :)

Reply:
That makes sense. Thanks for the additional context.